bin/_mb.bat

```diff
@@ -29,7 +29,7 @@ if "%PYBLISH_NUKE%"=="" goto :missing
 if "%PYBLISH_LITE%"=="" goto :missing
 if "%PYBLISH_QML%"=="" goto :missing
 if "%MINDBENDER_CORE%"=="" goto :missing
-:: if "%PYBLISH_USERNAME%"=="" goto :missing
+if "%PYBLISH_USERNAME%"=="" goto :missing
```

Comment:
> PUBLISH_USERNAME was suppose to be a static global variable set by the user
before they launched my.bat

Ok, better start an issue about it so we can talk about it and consider options, pros and cons.